prusa/connect/printer/__init__.py

```diff
@@ -682,6 +702,43 @@ def loop(self):
             except Exception:
                 log.exception("Unexpected exception caught in SDK loop!")
 
+    def snapshot_loop(self):
+        """Gets an item Snapshot from queue and sends it"""
+        while self.__running_loop:
+            try:
+                # Get the item to send
+                item = \
+                    self.snapshot_queue.get(timeout=const.TIMESTAMP_PRECISION)
+
+                # Send it
+                headers = self.make_headers(item.timestamp)
+                try:
+                    res = item.send(self.conn, self.server, headers)
+                    self.deduce_state_from_code(res.status_code)
```

Comment:
No! I told you, that we don't want set Error condition, when camera got 401 or 403 error. So Please delete whole deducate_state_from_code line.